gpa.py

```diff
@@ -147,11 +187,11 @@ def compute_results(student_records):
       "name": "James Webb",
       "id": "APT5005",
       "courses":[
-         {"title": "English", "unit": 2, "code": "ENG101", "score": 60, "term": "Fall"},
-         {"title": "Chemistry I", "unit": 4, "code": "CHE101", "score": 70, "term": "Fall"},
-         {"title": "Maths", "unit": 3, "code": "MTH101", "score": 80, "term": "Spring"},
-         {"title": "Chemistry II", "unit": 4, "code": "CHE102", "score": 91, "term": "Spring"}, 
-         {"title": "History", "unit": 2, "code": "HIS102", "score": 40, "term": "Spring"}
+         {"title": "English", "unit": 2, "code": "ENG101", "score": 60, "term": "Fall", "session":2020},
+         {"title": "Chemistry I", "unit": 4, "code": "CHE101", "score": 70, "term": "Fall", "session":2021},
+         {"title": "Maths", "unit": 3, "code": "MTH101", "score": 80, "term": "Spring", "session":2020},
+         {"title": "Chemistry II", "unit": 4, "code": "CHE102", "score": 91, "term": "Spring", "session":2021}, 
+         {"title": "History", "unit": 2, "code": "HIS102", "score": 40, "term": "Spring", "session":2020}
```

Comment:
I'd recommend to make session a string. So you should be using "2021" rather than a plain integer.

Reply:
Resolved